src/pages/policy.page.js

```diff
@@ -499,22 +503,27 @@ goToCurrentAdaptationPolicy () {
 
   async getTotalNumberOfRecordsOfPolicy() {
     const numberOfRecordsOfPolicy = await I.grabNumberOfVisibleElements(this.table.tableRows);
+    I.say('Rows count is '+numberOfRecordsOfPolicy)
     return numberOfRecordsOfPolicy
   },
 
   selectCountOfPolicies(itemCount) {
     const element = this.options.countOfPolicies;
-    I.selectOption(element, itemCount);
+    I.clickElement(element)
+    let countSelection = `//li[contains(.,'`+ itemCount +`')]`
```

Comment:
can use a template literal here

```suggestion
    let countSelection = `//li[contains(.,'${itemCount}')]`
```